### ISSUE TITLE
SpaceX

### ISSUE THREAD

**Issue by fboi94** [obrien66/Going-to-the-moon-hackwithIX2016#1]
Have you heard that space X might start allowing people to go to the moon?

**Comment by camranhansen** [comment]
Yeah, it's pretty cool. I hope to know someone who has traveled to the moon during my lifetime. But this repo is about data visualization for advertisement cpm in correlation to which region it is in, albeit with a deceptive name. We built this during a Hackathon using a really strange API, so it does not even visualize data well :(. But oh well.

**Comment by fboi94** [comment]
This project is still cool though! @camranhansen

**Comment by camranhansen** [comment]
Thanks man 💯
